api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -322,6 +329,19 @@ void shouldTestClaimBecomingExpiredAfter16WeeksPendingExpiry() throws JsonProces
         );
     }
 
+    //First argument is the previous cycle, second argument is the current cycle, third is the expected delivery date, fourth is the eligibility status.
+    private static Stream<Arguments> provideArgumentForClaimBecomingExpiredAfter16WeeksTest() {
+        return Stream.of(
+                Arguments.of(SINGLE_THREE_YEAR_OLD, emptyList(), DUE_DATE_IN_4_MONTHS, INELIGIBLE),
+                Arguments.of(emptyList(), SINGLE_THREE_YEAR_OLD, DUE_DATE_IN_4_MONTHS, INELIGIBLE),
+                Arguments.of(emptyList(), emptyList(), DUE_DATE_IN_4_MONTHS, INELIGIBLE),
+                Arguments.of(SINGLE_THREE_YEAR_OLD, emptyList(), NOT_PREGNANT, ELIGIBLE),
+                Arguments.of(SINGLE_THREE_YEAR_OLD, emptyList(), NOT_PREGNANT, INELIGIBLE),
+                Arguments.of(singletonList(CHILD_TURNED_FOUR_IN_LAST_CYCLE), emptyList(), NOT_PREGNANT, ELIGIBLE),
```

Comment:
Should `singletonList(CHILD_TURNED_FOUR_IN_LAST_CYCLE)` be in a constant l0kie SINGLE_CHILD_TURNED_FOUR_IN_LAST_CYCLE